backstop/tests/wizard.html

```diff
@@ -32,59 +32,59 @@ <h1>Wizard</h1>
     <h2 id="default">Default</h2>
 
     <section id="demo-default">
-      <div class="iui-wizards">
-        <div class="iui-wizards-wrapper">
```

Comment:
Now I am not sure about shortening names 😀 https://github.com/iTwin/iTwinUI/discussions/55

Reply:
That was my very first thought when I looked at this PR. 😄 

Not a fan of most of these new class names

Reply:
Class names are now a little more descriptive.

Reply:
Class names have been reverted back to the original, will do the class name refactor in #109 .